lib/trento_web/router.ex

```diff
@@ -85,16 +85,16 @@ defmodule TrentoWeb.Router do
 
     post "/collect", DiscoveryController, :collect
     post "/hosts/:id/heartbeat", HostController, :heartbeat
+
+    get "/settings", SettingsController, :settings
```

Comment:
My bad, I was probably misleading :sweat_smile: 
these should be in the previous one.

sorry.